### ISSUE TITLE
Make address optional if geoposition is provided

### ISSUE THREAD

**Issue by LuchtwachtersDelft** [opendata-stuttgart/devices.sensor.community#88]
If the geoposition is entered manually and the address omitted, the form refuses to submit because the address fields are required.

What is the use-case?

- user does not want to disclose their home address, let alone have it stored in a public database
- the sensor is not installed at a proper address, for example a lamp post, in a park or in a forest

**Comment by ricki-z** [comment]
The address won't be published. But in some cases (if the user has misplaced the pin on the map at registration) it could help us to get the correct GPS position.
It's possible to use a fantasy address, we won't check this. But in this case the user is responsible to choose the right GPS position.

**Comment by LuchtwachtersDelft** [comment]
Fantasy address is an acceptable workaround.

**Comment by ricki-z** [comment]
We don't write this on the registration page as it may 'confuse' some people. The are giving such a fantasy address and are wondering why the search won't find GPS coordinates (even if we mention this explicitly, this is called 'selective reading' :wink:).

**Comment by pjgueno** [comment]
@LuchtwachtersDelft It is always the same old thing: if people don't want to disclose the address, why do they want to participate in a project which try to promote commons,community, transparency and open data. They can build a sensor on their own and look at the values in their own house on their very screen and that's all.I assume that those people would be happy so. But it is not the goal of SC. I am sure it the same kind of people who don't accept the vaccination...
In conclusion it is imho really boring to always come back to this question: if you don't accept the conditions you must not participate.
